src/store/FilingHistoryList/actions.ts

```diff
@@ -0,0 +1,230 @@
+import { FilingSubTypes } from '@/enums'
```

Comment:
There isn't much to say about the actions in here. Except for the first one, they were methods in FHL and I moved them here.